core/common/src/main/java/alluxio/Constants.java

```diff
@@ -132,15 +132,19 @@
    */
   public static final int THRIFT_STOP_TIMEOUT_SECONDS = 60;
 
-  // ttl related
+  // Time-to-live
   public static final long NO_TTL = -1;
 
   // Security
-  // Security related constant value
   public static final int DEFAULT_FILE_SYSTEM_UMASK = 0022;
   public static final short DEFAULT_FILE_SYSTEM_MODE = (short) 0777;
   public static final short FILE_DIR_PERMISSION_DIFF = (short) 0111;
   public static final short INVALID_MODE = -1;
 
+  // Specific tier write
```

Comment:
Is there an explanation (in the code) of how these tier indexes are interpreted? (especially the negative offsets)

Also, do we want to refer to this type of value as an "offset", or something else other than"index"? "index" sounds like it can be used as an index of an array or list, but this value cannot be used like that.